src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -270,10 +270,13 @@ def __init__(self, config):
         if self.serverCert and self.serverKey and self.myproxySrv:
             self.proxy = self.setupMyProxy()
 
-        # Build a request string
-        self.reqStr = ('(OpSys == "LINUX" ) && (Arch == "INTEL" || Arch == "X86_64") '
-                       '&& stringListMember(GLIDEIN_CMSSite, DESIRED_Sites) '
-                       '&& ((REQUIRED_OS=="any") || stringListMember(GLIDEIN_REQUIRED_OS, REQUIRED_OS))')
+        # Build a request string.  All CMS resources match DESIRED_Sites on the START
```

Comment:
This plugin may soon be gone, so maybe put these comments on the other plugin instead.